tests/alpha.codegen.tests/resources/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.out
```

Comment:
This was added so when the generated code is compiled, git won't try to add the object files and executables.